Sources/CollectionViewModel.swift

```diff
@@ -155,59 +166,4 @@ public struct CollectionSectionViewModel: DiffableViewModel {
         self.footerViewModel = footerViewModel
         self.diffingKey = diffingKey
     }
-
-    private struct BlankSupplementaryViewModel: CollectionSupplementaryViewModel {
-        let height: CGFloat?
-        let viewInfo: SupplementaryViewInfo? = nil
-
-        func applyViewModelToView(_ view: UICollectionReusableView) { }
-    }
-}
-
-// MARK: Initializers without header/footer view models
-
-// Note: All initializers in this extension are undocumented, because we intend
-// to remove them. We want to get rid of the legacy functionality that creates
-// blank headers & footers instead of using spacing properties available via
-// `UICollectionViewLayout`s.
-extension CollectionSectionViewModel {
-
-    /// :nodoc:
-    public init(
-        cellViewModels: [CollectionCellViewModel],
-        headerHeight: CGFloat? = nil,
-        footerViewModel: CollectionSupplementaryViewModel? = nil,
-        diffingKey: String = UUID().uuidString) {
-        self.init(
-            cellViewModels: cellViewModels,
-            headerViewModel: BlankSupplementaryViewModel(height: headerHeight),
-            footerViewModel: footerViewModel,
-            diffingKey: diffingKey)
-    }
-
-    /// :nodoc:
-    public init(
-        cellViewModels: [CollectionCellViewModel],
-        headerViewModel: CollectionSupplementaryViewModel? = nil,
-        footerHeight: CGFloat? = nil,
-        diffingKey: String = UUID().uuidString) {
-        self.init(
-            cellViewModels: cellViewModels,
-            headerViewModel: headerViewModel,
-            footerViewModel: BlankSupplementaryViewModel(height: footerHeight),
-            diffingKey: diffingKey)
-    }
-
-    /// :nodoc:
-    public init(
-        cellViewModels: [CollectionCellViewModel],
-        headerHeight: CGFloat? = nil,
-        footerHeight: CGFloat? = nil,
-        diffingKey: String = UUID().uuidString) {
-        self.init(
-            cellViewModels: cellViewModels,
-            headerViewModel: BlankSupplementaryViewModel(height: headerHeight),
-            footerViewModel: BlankSupplementaryViewModel(height: footerHeight),
-            diffingKey: diffingKey)
-    }
```

Comment:
These were causing ambiguous naming errors in Swift (probably because of my diffing key changes)

Per the comment above, these needed to be removed anyway. 

We don't use them in PlanGrid either, so 👌